src/ring_theory/ideal_operations.lean

```diff
@@ -474,8 +481,76 @@ le_antisymm (map_le_iff_le_comap.2 $ mul_le.2 $ λ r hri s hsj,
   by rw [← f.map_mul];
   exact mem_map_of_mem (mul_mem_mul hri hsj))
 
+@[simp] lemma map_id : I.map (ring_hom.id R) = I :=
+by ext; simp [ideal.map]
+
+lemma map_map {T : Type*} [comm_ring T] {I : ideal R} (f : R →+* S)
+  (g : S →+*T) : (I.map f).map g = I.map (g.comp f) :=
```

Comment:
If you prove `gc_map_comap` and `comap_comap` first (which should be basically `rfl`) then this would follow by uniqueness of "adjoints" (`galois_connection l u -> galois_connection l' u -> l = l'`) though I'm not sure we have this lemma either.

Reply:
Very nice. This has now been done.